src/core/render-core.js

```diff
@@ -38,13 +50,15 @@ async function render(_opts = {}) {
   logOpts(opts);
 
   const browser = await puppeteer.launch({
-    headless: !config.DEBUG_MODE,
-    ignoreHTTPSErrors: opts.ignoreHttpsErrors,
+    headless: opts.headless || !config.DEBUG_MODE,
+    ignoreHTTPSErrors: opts.ignoreHttpsErrors || config.IGNORE_HTTPS_ERRORS,
     args: ['--disable-gpu', '--no-sandbox', '--disable-setuid-sandbox'],
     sloMo: config.DEBUG_MODE ? 250 : undefined,
   });
   const page = await browser.newPage();
 
+  if (opts.needAuth) await getAuth(page, opts.username, opts.passwd);
```

Comment:
We need a way to not have to navigate to login page and enter user/pass every time we need to print a page. Could we provision a way to stay logged in for certain amount of time for a special user? I am ok with this for the MVP but we really need a better solution before shipping this

cc. @jpang-h4